### PR TITLE
updating the external documentation

### PR DESCRIPTION
**Pull request by morten-skelmose**
Ticket: https://voidev.atlassian.net/browse/RID-1654

Adding MDS 2.0 to the documentation

### CODE REVIEW

**Review by morten-skelmose — COMMENTED** [review]
deploy to stage please

File: maas-light.md [hunk 1]
```diff
@@ -6,9 +6,416 @@ menu: main
...
+
+
+## Trips
+
+> A trips request.
+
+```shell
+# v2.0
+$ curl -H "Accept: application/vnd.mds+json;version=2.0"
+  api.voiapp.io/v1/partner-apis/mds/1/trips?end_time=2024-06-13T10
```
Comment (erichall) [inline maas-light.md:71]: Maybe you could explain what a trip is very short and then state the url like
A trip is a trip, you can get trips on this path:
/v1/partner-apis/mds/{zone-id}/trips?end_time={time}

example:
your amazing curl :)

File: maas-light.md [hunk 2]
```diff
@@ -6,9 +6,416 @@ menu: main
...
+            "trip_type": [
+                "rider"
+            ]
+        }
+    ]
+}
+```
+
+
+A trip represents a journey taken by a customer with a geo-tagged start and stop point.
```
Comment (erichall) [inline maas-light.md:114]: ah here it was, maybe move this up before the example?

File: maas-light.md [hunk 3]
```diff
@@ -6,9 +6,416 @@ menu: main
...
+The `/trips endpoint allows a user to query historical trip data that happened within the queried hour.
+
+See Open Mobility Foundation's MDS description on trips [v2.0](https://github.com/openmobilityfoundation/mobility-data-specification/blob/release-2.0.0/provider/README.md#trips).
+
+### HTTPS request
+
+`GET api.voiapp.io/v1/partner-apis/mds/{zoneID}/trips`
```
Comment (erichall) [inline maas-light.md:121]: AND HERE!!!!! :D

Comment (erichall) [reply maas-light.md:121]: yeah maybe just move the example to the end ?? :)

File: maas-light.md [hunk 4]
```diff
@@ -6,9 +6,416 @@ menu: main
...
+## Vehicles
+
+> A vehicles request.
+
+```shell
+# v2.0 - All vehicles
+$ curl -H "Accept: application/vnd.mds+json;version=2.0"
+  api.voiapp.io/v1/partner-apis/mds/1/vehicles
+
+# V2.0 - Single device
```
Comment (erichall) [inline maas-light.md:144]: device = vehicle ?

**Review by erichall — APPROVED** [review]

**Pull request merged**